### PR TITLE
fix: Flicking is animating on moveEnd event

### LINKED ISSUE #541: Flicking is animating on moveEnd event
## Description
- https://github.com/naver/egjs-flicking-plugins/issues/26

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#541

## Details
This fixes #541

### CODE REVIEW

File: src/camera/Camera.ts [hunk 1]
```diff
@@ -271,11 +271,11 @@ abstract class Camera {
    * <ko>{@link ERROR_CODE NOT_ATTACHED_TO_FLICKING} {@link Camera#init init}이 이전에 호출되지 않은 경우</ko>
    * @return {this}
    */
-  public async lookAt(pos: number): Promise<void> {
+  public lookAt(pos: number): void {
     const prevPos = this._position;
 
     this._position = pos;
-    await this._refreshVisiblePanels();
+    this._refreshVisiblePanels();
```
Comment (mixed) [inline src/camera/Camera.ts:278]: The logic seems to be different from the previous one. I wonder if that was part of the intention. The point where EVENTS.VISIBLE_CHANGE occurs after flicking.renderer.render() is finished is the same as before. However, the three methods below can be executed before VISIBLE_CHANGE occurs differently from the existing ones. I wonder if there are any problems with this part.
```js
this._checkNeedPanel();
this._checkReachEnd(prevPos, pos);
this._applyTransform();
```

**Review by mixed — COMMENTED** [review]
If there are no problems with the `lookat` review, this is a good enough PR.

File: src/camera/Camera.ts [hunk 2]
```diff
@@ -526,13 +526,13 @@ abstract class Camera {
...
 
     if (added.length > 0 || removed.length > 0) {
-      await flicking.renderer.render();
-
-      flicking.trigger(new ComponentEvent(EVENTS.VISIBLE_CHANGE, {
-        added,
-        removed,
-        visiblePanels: newVisiblePanels
-      }));
+      void flicking.renderer.render().then(() => {
```
[the remark this answers is not in the document]
Comment (WoodNeck) [reply src/camera/Camera.ts:529]: That's basically due to our eslint setting

File: src/camera/Camera.ts (hunk 1 above)
Comment (WoodNeck) [reply src/camera/Camera.ts:278]: Yes, the timing changed and even the interface of the internal component is changed.
But as there wasn't any previous definition of the timing of `visibleChange`, I hope that's okay.

**Review by mixed — APPROVED** [review]
LGTM

**Review by daybrush — APPROVED** [review]
LGTM

**Pull request merged**